docs/config.ini

```diff
@@ -0,0 +1,77 @@
+[project]
+
+# The project name, used as the filename of the package and the PDF file. For
+# example, if set to d2l-book, then will build d2l-book.zip and d2l-book.pdf
+name = autogluon
```

Comment:
shall we use `autogluon.cloud` here?

Reply:
No, the website we built follow the same metadata as the main autogluon website.
The only thing we need is the cloud page, but it requires the same metadata to avoid conflict when copied over